.gitlab-ci.yml

```diff
@@ -0,0 +1,43 @@
+image: ubuntu:18.04
```

Comment:
Can "image" be set directly to `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.12`?

Reply:
It can.  Will change.